sccp/figures/figureS1.py

```diff
@@ -21,18 +21,27 @@ def makeFigure():
     # Add subplot labels
     subplotLabel(ax)
 
-    rank = 30
+    rank = 5
 
     (
         lupus_tensor,
-        _,
-        row_colors,
-    ) = load_lupus_data()  # don't need to grab cell types here
+        obs,
+    ) = load_lupus_data(every_n = 10)  # so that it builds quickly
+
 
     weights, factors, _, _ = parafac2_nd(
         lupus_tensor, rank=rank, n_iter_max=20, random_state=1
     )
 
+    status = (
+        obs[["sample_ID", "SLE_status"]]
+        .sort_values(by="sample_ID")
```

Comment:
@pascoesean isn't this switching the order that Pf2 orders the patients?

Reply:
Yes, good catch thank you (I think this is holdover from when I was trying to do something in a separate notebook). It shouldn't make a difference in output though (phew) since the observations are already *mostly* ordered by the scRNA import function itself. I don't see a need to keep this though so I'll take it out

Reply:
Yeah, we just want to make sure we are not mixing the order of the patients' statuses